docs/user-manual/onboarding-guide/programguide

```diff
@@ -0,0 +1,112 @@
+            - [ ] Find translation resources and add time for translations to your project timeline.  
+      - [ ] Data validation: Implement program checks (such as phone number formats and hints to users “select only two choices” to prevent errors and ensure data accuracy.  
+      - [ ] Work with your IT team to select an identity and authentication provider (i.e. Login.gov).  
+- [ ] Identify the list of relevant Key Performance Indicators (KPIs) and metrics to identify what a successful launch would look like for each program launch. Example metrics:  
+      - [ ] Reduction in time to complete an application.  
+      - [ ] Increase in residents’ access to annual savings / benefits (dollars).  
+      - [ ] Increase in number of programs online / added to CiviForm.  
+- [ ] Identify key stakeholders throughout the community (eg. CBOs) and local governments who might be able to connect target user groups with the program to ensure a successful launch.  
+- [ ] If resident applications are managed or reviewed in an external case management system, work with your technical team to ensure necessary data pipelines are set up and tested.  
+- [ ] Develop a plan for thorough testing: Conduct comprehensive testing to identify and fix issues before launch.  
+- [ ] Train program administrators how to use CiviForm and how to route questions / issues they have.   
+- [ ] Perform user experience studies.  
+      - [ ] Ensure that at least 10 real end users have tested the new program.  
+      - [ ] Work with specific programs to test the end-to-end process for application submission and processing and make adjustments as needed.  
+- [ ] Pilot program: Plan a pilot launch to gather feedback and make improvements.  
+- [ ] Decide on launch announcement plans to ensure that residents can find and apply for the new streamlined programs.  
+      - [ ] Identify target audience/user group, communication channels (eg. press release, website announcement, social media posts, a newsletter) to share the news with residents.  
+      - [ ] Develop a communication plan: Inform citizens and stakeholders about the new system and its benefits and consider how you may drive awareness about the program.  
+      - [ ] Consider relevant metrics to measure if you are reaching your residents.  
+- [ ] Generate excitement by recording demos and sharing any initial findings, positive data outcomes, or testimonials resulting from use of CiviForm to connect users to the program.   
+- [ ] **Announce CiviForm to your community**   
+- [ ] Put ample opportunities for feedback in place during and after the launch so that users can flag any questions or bugs.   
+- [ ] If programs will be rolled out in phases, provide users with a timeline of future planned launches for additional programs.
+
+## Next steps \- Ongoing Evaluation and Improvement
```

Comment:
**question:** What does \- do for formatting?

Reply:
The dash and [ ] is the markdown code for checkbox. Google Docs outputted this and it was the same as the checkboxes that you pointed me to. However, it doesn't display correctly on the website (maybe due to the layout and css), but does seem to display correctly on the docs site. I wanted to check this in to the docs site to see how it behaves.